openssl/src/ssl/tests.rs

```diff
@@ -344,19 +465,20 @@ run_test!(get_peer_certificate, |method, stream| {
 #[test]
 #[cfg(feature = "dtlsv1")]
 fn test_write_dtlsv1() {
-    let sock = UdpSocket::bind("127.0.0.1:0").unwrap();
-    let stream = sock.connect("127.0.0.1:15410").unwrap();
+    let yes = iter::repeat("y\n").take(1_000_000).collect::<String>();
```

Comment:
I'm not sure why `yes` was piped in the first place, since this test doesn't actually read anything. Do things still work if you cut that part out entirely?